src/content/community/support/index.md

```diff
@@ -38,8 +38,11 @@ Looking for an Ethereum wallet? [Explore our full list of Ethereum wallets](/wal
 
 Building can be hard. Here are some development focused spaces with experienced Ethereum developers that are happy to help.
 
+- [Alchemy University](https://university.alchemy.com/#starter_code)
```

Comment:
This looks fine as a link to me. I am just wondering out of curiosity, how is the Discords managed for Alchemy University? I see you need an NFT for access to the channel, is this free?

Reply:
Great question! Re: Discord - we have an education team (eng, product, devrel, etc) at Alchemy that created Alchemy University and helps man the Discord along with some of our community ambassadors. Re: free - Alchemy University like all our dev content is completely free - the NFT drops were to give people early access when the course launches in a few weeks. We have a waitlist of tens of thousands which has been so awesome to see. Lmk if you have any more questions! We are definitely doing a lot of Ethereum specific content so if there's any way we can be helpful to Ethereum too, seriously pls just lmk :)